iceoryx_posh/test/integrationtests/service_discovery/test_roudi_findservice.cpp

```diff
@@ -40,6 +40,70 @@ TEST_F(RoudiFindService_test, OfferSingleMethodServiceSingleInstance)
     ASSERT_THAT(*instanceContainer.begin(), Eq(IdString("instance1")));
 }
 
+TEST_F(RoudiFindService_test, OfferServiceWithDefaultDescription)
+{
+    senderRuntime->offerService(iox::capro::ServiceDescription());
```

Comment:
We try to write our test according to the AAA pattern. Please also do it that way that makes it easier for all the others
[AAA pattern](https://blog.ploeh.dk/2013/06/24/a-heuristic-for-formatting-code-according-to-the-aaa-pattern/)

[iceoryx example](https://github.com/eclipse/iceoryx/blob/master/iceoryx_posh/test/moduletests/test_popo_publisher_port.cpp#L134-L148)

Reply:
done